feelslikemonday/app/src/androidTest/java/com/example/feelslikemonday/DAO/FollowPermissionDAOTest.java

```diff
@@ -51,7 +51,71 @@ public void onCallback() {
                 signal.countDown();
             }
         });
+
         signal.await();
+    }
+
+    @Test
+    public void updateObject() throws InterruptedException {
+        final CountDownLatch signal = new CountDownLatch(1);
+        final FollowPermission followPermission = new FollowPermission("uTEST-bill");
 
```

Comment:
Just confirming that you are sure this user is a new user and is compatible with the latest version of the db

Reply:
The new tests have been confirmed to pass. FollowPermissions can exist without users (though it doesn't make sense), so these tests work with the db.